packages/blade/src/components/SkipNav/__tests__/SkipNav.web.test.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import userEvents from '@testing-library/user-event';
+import renderWithTheme from '../../../_helpers/testing/renderWithTheme.web';
+import { SkipNavContent, SkipNavLink } from '../SkipNav.web';
+
+describe('<SkipNav />', () => {
+  it('<SkipNavLink /> should render correctly', () => {
+    const { getByRole } = renderWithTheme(<SkipNavLink />);
+
+    expect(getByRole('link')).toHaveTextContent('Skip to content');
+  });
+
+  it('<SkipNavLink /> should render correctly with children', () => {
+    const { getByRole } = renderWithTheme(<SkipNavLink children="Skip" />);
```

Comment:
why children as a prop? and not `<SkipNavLink>Skip</SkipNavLink>`?

Reply:
Both are same thing tho. children is a prop too. I just like the self closing jsx for simple components while testing.